packages/devtools-plugin/src/index.ts

```diff
@@ -31,9 +39,9 @@ const devToolsPlugin = <State>(): ProdoPlugin<
       const message: DevMessage = {
         destination: "devtools",
         type: "completedEvent",
-        contents: { event },
+        contents: { event: serialize(event) },
```

Comment:
Why serialize here? The message is going to be serialized anyway isn't it?

Reply:
Oh, forgot to remove that after changing the code to serialize the whole message. Oops.